cloud/tasks/persistence/s3_metrics.go

```diff
@@ -72,13 +74,18 @@ func (m *s3Metrics) StatCall(
 
 			errorCounter.Inc()
 
-			if errors.Is(*err, context.DeadlineExceeded) {
-				timeoutCounter.Inc()
+			dmError := (*err).(*errors.RetriableError)
+			if aerr, ok := dmError.Unwrap().(awserr.Error); ok {
```

Comment:
Call `As` instead of Unwrap https://github.com/ydb-platform/nbs/blob/main/cloud/tasks/errors/errors.go#L355